.github/CONTRIBUTING.md

```diff
@@ -70,16 +70,7 @@ Contributions are welcome! Here's how you can help:
 ### Important note about automated GitHub checks
```

Comment:
This whole section was added in fc2e120b03e65feb9ccfb92f9d3b68fda230bc7b because of clang-format. I think it should be removed completely.

Reply:
Done.